docs/reference-guides/data/data-core.md

```diff
@@ -179,6 +248,31 @@ Returns the latest autosaves for the post.
 
 May return multiple autosaves since the backend stores one autosave per author for each post.
 
+_Usage_
+
+```js
+import { store as coreDataStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { sprintf, __ } from '@wordpress/i18n';
```

Comment:
```suggestion
import { sprintf } from '@wordpress/i18n';
```

It doesn't look like `__` import is actually needed here based on the code below. I noticed some similar issues in the other examples. I'd double-check that all the listed imports are needed.

Reply:
Nice catch! I've updated and adding some missing dependencies on the useSelect calls